tests-core/src/main/java/com/pragmaticobjects/oo/tests/bdd/BehaviorAssertion.java

```diff
@@ -0,0 +1,29 @@
+package com.pragmaticobjects.oo.tests.bdd;
+
+import com.pragmaticobjects.oo.tests.Assertion;
+
+public final class BehaviorAssertion<S, R> implements Assertion {
```

Comment:
[oo-atom](https://github.com/pragmatic-objects/oo-atom) prohibits final classes for [certain](https://pragmaticobjects.com/chapters/002_never_make_class_final.html) [reasons](https://pragmaticobjects.com/chapters/005_implementation_inheritance_paranoia.html). Let the class being open and finalize its methods instead.